jenkins-scripts/dsl/brew_release.dsl

```diff
@@ -121,12 +120,7 @@ release_job.with
 // -------------------------------------------------------------------
 // 2. BREW bottle creation MATRIX job from pullrequest
 def bottle_job_builder = matrixJob(bottle_builder_job_name)
-// set enable_github_pr_integration flag to false so we can customize trigger behavior
-OSRFBrewCompilationAnyGitHub.create(bottle_job_builder,
-                                    "osrf/homebrew-simulation",
```

Comment:
does it work without `OSRFBrewCompilationAnyGitHub` and the `"osrf/homebrew-simulation"` parameter? I'm not sure how it would attach to pull requests without that configuration

Reply:
good catch, changes are broken indeed. I tried to solve them in c7053c3. The new diff seems much better:

```diff
--- generic-release-homebrew_triggered_bottle_builder.xml	2021-10-06 17:00:03.366135806 +0200
+++ /tmp/generic-release-homebrew_triggered_bottle_builder.xml	2021-10-06 16:59:21.924941433 +0200
@@ -261,37 +261,6 @@
             <saveOutput>false</saveOutput>
             <disabled>false</disabled>
         </hudson.plugins.emailext.ExtendedEmailPublisher>
-        <io.jenkins.plugins.analysis.core.steps.IssuesRecorder>
-            <analysisTools>
-                <io.jenkins.plugins.analysis.warnings.Clang>
-                    <id></id>
-                    <name></name>
-                    <pattern></pattern>
-                    <reportEncoding></reportEncoding>
-                    <skipSymbolicLinks>false</skipSymbolicLinks>
-                </io.jenkins.plugins.analysis.warnings.Clang>
-            </analysisTools>
-            <sourceCodeEncoding></sourceCodeEncoding>
-            <ignoreQualityGate>false</ignoreQualityGate>
-            <ignoreFailedBuilds>true</ignoreFailedBuilds>
-            <referenceJobName></referenceJobName>
-            <healthy>0</healthy>
-            <unhealthy>0</unhealthy>
-            <minimumSeverity>
-                <name>LOW</name>
-            </minimumSeverity>
-            <filters></filters>
-            <isEnabledForFailure>false</isEnabledForFailure>
-            <isAggregatingResults>false</isAggregatingResults>
-            <isBlameDisabled>false</isBlameDisabled>
-            <qualityGates>
-                <io.jenkins.plugins.analysis.core.util.QualityGate>
-                    <threshold>1</threshold>
-                    <type>TOTAL</type>
-                    <status>WARNING</status>
-                </io.jenkins.plugins.analysis.core.util.QualityGate>
-            </qualityGates>
-        </io.jenkins.plugins.analysis.core.steps.IssuesRecorder>
         <hudson.tasks.ArtifactArchiver>
             <artifacts>pkgs/*</artifacts>
             <allowEmptyArchive>true</allowEmptyArchive>
```